.gitignore

```
@@ -10,13 +10,13 @@ __pycache__
 *.c
 
 # Other generated files
-*/version.py
 */cython_version.py
 htmlcov
 .coverage
 MANIFEST
 .ipynb_checkpoints
 example_notebooks/test.png
+*/version.py
```

I found myself also have to ignore `pip-wheel-metadata/` for my packages. 🤷‍♀ 
```suggestion
*/version.py
pip-wheel-metadata/
```